src/main/java/frc/trigon/robot/commands/factories/AutonomousCommands.java

```diff
@@ -37,20 +38,23 @@ public static Command getResetPoseToAutoPoseCommand(Supplier<String> pathName) {
                     if (DriverStation.isEnabled())
                         return;
                     final Pose2d autoStartPose = PathPlannerAuto.getStaringPoseFromAutoFile(pathName.get());
-                    RobotContainer.POSE_ESTIMATOR.resetPose(new MirrorablePose2d(autoStartPose, true).get());
+                    RobotContainer.POSE_ESTIMATOR.resetPose(autoStartPose);
```

Comment:
This might need to be a mirrorable pose (might be the cause of that problem we had when we used the odometry for auto where it was the flipped pose)